src/containers/BooksForm.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-restricted-syntax */
+import React, { Component } from 'react';
+
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bookGenres: ['Action and Adventure', 'Anthology', 'Chick lit', 'Crime', 'Drama', 'Fantasy'],
+    };
+  }
+
+  render() {
+    const { bookGenres } = this.state;
```

Comment:
Remove this line as your genres will not be coming from the `state`. Implement my above point instead and also write these lines of code to map all the genres to display in the select options:

```
const bookGenres = ['Action and Adventure', 'Anthology', 'Chick lit', 'Crime', 'Drama', 'Fantasy'];

const bookOptions = bookGenres.map(category => <option key={category} value={category}>{category}</option>);

```